docker/start.py

```diff
@@ -131,10 +136,10 @@ def generate_config_from_template(
 
     if ownership is not None:
         log(f"Setting ownership on /data to {ownership}")
-        subprocess.check_output(["chown", "-R", ownership, "/data"])
+        subprocess.run(["chown", "-R", ownership, "/data"])
```

Comment:
You need to pass `check=True` to `run()` or else we will silently ignore commands that fail.

See the docs for check_output and check_call.

Reply:
Just to be clear, all of them? Or just those like above where it accesses outside OS commands? There are a total of 4 between both files.

Reply:
All of them---we want errors to be reported loudly (as is the case with `check_output` and `check_call`).

Reply:
I guess they all sort of do, don't they.